Makefile

```diff
@@ -40,11 +40,26 @@ all build:
 #
 # Example:
 #   make check
-#   make test
 #   make check WHAT=pkg/build GOFLAGS=-v
-check test:
+check:
 	hack/test-go.sh $(WHAT) $(TESTS)
-.PHONY: check test
+.PHONY: check
+
+# Build and run the complete test-suite.
+#
+# Args:
+#   GOFLAGS: Extra flags to pass to 'go' when building.
+#
+# Example:
+#   make test
+#   make test GOFLAGS=-v
+test: export KUBE_COVER= -cover -covermode=atomic
```

Comment:
You actually can't use quotes in make file in this syntax.


> On Jan 19, 2015, at 8:32 AM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In Makefile:
> 
> >  	hack/test-go.sh $(WHAT) $(TESTS)
> > -.PHONY: check test
> > +.PHONY: check
> > +
> > +# Build and run the complete test-suite.
> > +#
> > +# Args:
> > +#   GOFLAGS: Extra flags to pass to 'go' when building.
> > +#
> > +# Example:
> > +#   make test
> > +#   make test GOFLAGS=-v
> > +test: export KUBE_COVER= -cover -covermode=atomic
> do you need quotes around the value since it has spaces?
> 
> —
> Reply to this email directly or view it on GitHub.
>